cmd/preflight/cmd/list_checks.go

```diff
@@ -38,6 +38,7 @@ func printChecks(w io.Writer) {
 		"automatically applied for container checks if preflight determines a scratch exception flag has been added to your Red Hat Connect project"))
 	fmt.Fprintln(w, formattedPolicyBlock("Container Scratch (Root) Exception", engine.ScratchRootContainerPolicy(context.TODO()),
 		"automatically applied for container checks if preflight determines scratch and root exception flags have both been added to your Red Hat Connect project"))
+	// todo-adam should we add konflux here?
```

Comment:
My gut says no, but I'm open to the idea.

Reply:
My gut said no as well, we'll let others chime in.

Reply:
I removed this todo, and we'll leave this out of the `list-checks` cmd.